File: codewars/ruby/lib/multiple_of_3_or_5.rb
class Multiple
  def solution(number)
    num_range = Array(0..(number -1))
    #mult = num_range.map { |x| x if x % 3 == 0 || x % 5 == 0 }.compact
    mult = num_range.map { |x| x if (x % 3).zero? || (x % 5).zero? }.compact
    mult.sum
  end
end

# https://www.codewars.com/kata/514b92a657cdc65150000006/train/ruby

File: codewars/js/lib/multiple_of_3_or_5.js
function solution(number) {
  let numbers = [];
  for (const x of Array(number).keys()) { (x % 3 === 0 && numbers.push(x)) || (x % 5 === 0 && numbers.push(x))}
  return numbers.reduce((a, b) => a + b, 0)
}
module.exports = solution;

